WcaOnRails/app/controllers/polls_controller.rb

```diff
@@ -70,6 +70,6 @@ def poll_params
     if params[:commit] == "Confirm" && current_user.can_create_poll?
       poll_params[:confirmed_at] = Time.now
     end
-    return poll_params
+    poll_params
```

Comment:
I have little opinion on this. I know there are dragons with procs and returns, and I'm okay with lowering the probability I discover those dragons.